README.md

```diff
@@ -86,6 +86,15 @@ Location of created sqlite database must be known:
 export "BKMR_DB_URL=db-path"
 ```
 
+You can set (not required) this variable to change `--fzf` mode default options:
+```bash
+export "BKMR_FZF_OPTS=--reverse --height 20% --show-tags"
+```
+Currently available flags: 
+- `--reverse` (defaults to false)
+- `--height` (defaults to 50%)
```

Comment:
this parameter does seem to make a big difference on my machine. Not sure what to expect here. Do you mind give a brief explanation of the to be expected effect?

Reply:
If I'm right, `--height` is a skim option to declare how much height on your terminal the skim output should have (in percent).

 `--reverse` is a skim option to declare that the skim search-input should be on top of the skim output.
 
 Should we explain this in the readme too ? 
 (I'm a user of the classic fzf tool, from which skim was inspired, so to me it was clear enough)

Reply:
You are right, I also understand this from the original fzf. However, on my machine when I was playing around with it, it did not really make a big difference (sorry for the typo in my original comment: did NOT make a big difference). 

Have you tested it and does it make a difference for you?

Reply:
Yes, it works well for me. 
I always use my terminal (alacritty) in full-screen.

For instance `BKMR_FZF_OPTS="--height 10%"`
will narrow the height of skim results on my terminal, allowing me to scroll up/down if I need.

But `BKMR_FZF_OPTS="--height 100%" ` will take the whole screen.

Note that I left the default to "50%", as it was in the code initially :
`BKMR_FZF_OPTS="--reverse` will only reverse the input field on the top of the results, displaying them with the default height of 50%.